src/constants.ts

```diff
@@ -68,7 +68,8 @@ export const DEFAULT_OPTIONS = {
   pages: {},
   skipSettingLocaleOnNavigate: false,
   types: 'composition',
-  debug: false
+  debug: false,
+  parallelPlugin: false
```

Comment:
Oh, I see. That's instresting!
https://github.com/nuxt/nuxt/pull/20460

When adding this option, we need to add docs about the option.
Can you also write it to docs?

Reply:
nvm i'm blind :joy: .... there's indeed a doc for parallel plugins